src/azure-cli/HISTORY.rst

```diff
@@ -23,6 +23,10 @@ Release History
 
 * az network private-dns link vnet create/update: Fixes #9851. Support cross-tenant virtual network linking.
 
+**Storage**
+
+* Adds support for WebAssembly (.wasm) mimetype detection
```

Comment:
LGTM. Can you change 'Adds' to 'Add' here?

Reply:
Done!